core/kazoo_modb/priv/couchdb/views/mailbox_messages.json

```diff
@@ -2,15 +2,70 @@
     "_id": "_design/mailbox_messages",
     "language": "javascript",
     "views": {
+        "count_per_box_folder": {
+            "map": [
+                "function(doc) {",
+                "  if (doc.pvt_type != 'mailbox_message' || doc.pvt_deleted)",
+                "    return;",
+                "  emit([doc.source_id, doc.metadata.folder, doc.metadata.timestamp]);",
+                "}"
+            ]
+        },
         "count_per_folder": {
-            "map": "function(doc) { if (doc.pvt_type != 'mailbox_message' || doc.pvt_deleted) return; emit([doc.source_id, doc.metadata.folder], 1); }",
-            "reduce": "_count"
+            "map": [
+                "function(doc) {",
+                "  if (doc.pvt_type != 'mailbox_message' || doc.pvt_deleted)",
+                "    return;",
+                "  emit([doc.metadata.folder, doc.metadata.timestamp, doc.source_id]);",
```

Comment:
does `emit/1` use `null` as the value by default?

Reply:
yes, though i'm not sure about bigcouch :-)

Reply:
well I see karl added `missing_metadata` without specifying value, so it is working in bigcouch too.